sunscreen_runtime/src/runtime.rs

```diff
@@ -158,6 +158,22 @@ impl Runtime {
         Ok(keys)
     }
 
+    /**
+     * Takes a byte array of the serialized private key material and returns a PrivateKey instance
+     */
+    pub fn bytes_to_private_key(&self, sk: &[u8]) -> Result<PrivateKey> {
```

Comment:
We shouldn't have a special function for deserialization, but should use serde's `Deserialize` trait as all the other keys to. `WithContext` is the key to bridging SEAL's serialization with serde.